packages/evm/README.md

```diff
@@ -59,53 +70,41 @@ evm
 This projects contain the following examples:
 
 1. [./examples/decode-opcodes](./examples/decode-opcodes.ts): Decodes a binary EVM program into its opcodes.
+1. [./examples/run-code-browser](./examples/run-code-browser.js): Show how to use this library in a browser.
 
 All of the examples have their own `README.md` explaining how to run them.
 
 # API
 
-## VM
-
-For documentation on `VM` instantiation, exposed API and emitted `events` see generated [API docs](./docs/README.md).
-
-## VmState
-
-The VmState is the wrapper class that manages the context around the underlying state while executing the VM like `EIP-2929`(Gas cost increases for state access opcodes). A Custom implementation of the `StateManager` can be plugged in the VmState
+## EVM
 
-# BROWSER
+For documentation on `EVM` instantiation, exposed API and emitted `events` see generated [API docs](./docs/README.md).
 
-To build the VM for standalone use in the browser, see: [Running the VM in a browser](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/vm/examples/run-code-browser.js).
+## VM/EVM Relation
 
-# SETUP
+This package contains the inner Ethereum Virtual Machine core functionality which was included in the [@ethereumjs/vm](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/vm) package up till v5 and has been extracted along the v6 release.
 
-## Chain Support
+This will make it easier to customize the inner EVM, which can now be passed as an optional argument to the outer `VM` instance.
 
-Starting with `v5.1.0` the VM supports running both `Ethash/PoW` and `Clique/PoA` blocks and transactions. Clique support has been added along the work on PR [#1032](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1032) and follow-up PRs and (block) validation checks and the switch of the execution context now happens correctly.
+At the moment the `EVM` package can not be run standalone and it is therefore recommended for most use cases to rather use the `VM` package and access `EVM` functionality through the `vm.evm` property.
 
-### Ethash/PoW Chains
+## Execution Environment (EEI) and State
 
-`@ethereumjs/blockchain` validates the PoW algorithm with `@ethereumjs/ethash` and validates blocks' difficulty to match their canonical difficulty.
+For the EVM to properly work it needs access to a respective execution environment (to e.g. request on information like block hashes) as well as the connection to an outer account and contract state.
 
-### Clique/PoA Chains
+To ensure a unified interface the `EVM` provides a TypeScript `EEI` interface providing which includes the necessary function signatures for access to environmental parameters as well as the VM state.
```

Comment:
Thanks, have added these tasks to the newly revived "Tiny Fixes Meta Issue" issue #2032 🙂, since otherwise too small left-overs to address separately.